src/index.tsx

```diff
@@ -1112,3 +1112,5 @@ export const Drawer = {
   Title: DialogPrimitive.Title,
   Description: DialogPrimitive.Description,
 };
+
+export { DrawerContext, useDrawerContext };
```

Comment:
Why do we need to export `DrawerContext`? This can't be used externally.

Reply:
@emilkowalski Hi, Is it any way to get access to context to close drawer?